### PR TITLE
Petras portfolio

### PR DESCRIPTION
**Pull request by petrasson**
This project was about building a portfolio to present yourself, your work and your skills.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+        <div class="box-wrapper" role ="layout">
+          <div class="box-title" role ="layout">
+            <h2 class="title" tabindex="2">TECH</h2></div>
+          <div class="box" role ="layout">
+            <p tabindex="3">
+            HTML, CSS, Flexbox, JavaScript, ES6, JSX, React, Web Accessibility, API:s, mob-programming, pair-programming, Github.
+            </p>
+          </div>
+      </section>
+      <section class="project-container">
```
Comment (agadman) [inline index.html:55]: Nice job with the featured project section. Maybe you can link the projects to your Github repos later on?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+            <h2 class="title" tabindex="37">FOR MORE</h2></div>
+            <div class="grid-box-icons" role ="layout">
+            <div class="icons-end" role="link" tabindex="38"><img class="icon-picture" src="./icons/linkedin_logotype.svg" alt="Go to Petras LinkedIn"><h2 class="project-title"><a href="https://www.linkedin.com/in/petra-eriksson-623b6a163/">LINKEDIN</a></h2>
+            </div>
+            <div class="icons-end" role="link" tabindex="39"><img class="icon-picture" src="./icons/github_logotype.svg" alt="Go to Petras Github"><h2 class="project-title"><a href="https://github.com/petrasson">GITHUB</a></h2></div>
+          </div>
+        </div>
+      </section>
+
+    <!-- Footer -->
```
Comment (agadman) [inline index.html:249]: It's great with the clear comments you make that separate the different sections. E.g.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+          </div>
+      </section>
+      <section class="project-container">
+        <div class="box-wrapper" role ="layout" tabindex="3">
+          <div class="box-title" role ="layout">
+            <h2 class="title" tabindex="4">FEATURED PROJECTS</h2>
+          </div>
+          <div class="project-box" role ="layout">
+            <div class="project" role ="layout">
+            <div class="project-photo" role ="photo" style="background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('./Images/weatherapp_picture.png');">
```
Comment (agadman) [inline index.html]: Nice styling of the images! It looks good.
Maybe you can move the styling of them to the css later on?

File: style.css [hunk 4]
```diff
@@ -0,0 +1,570 @@
...
+
+.header-image {
+  background-color: #7d6081;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.header-wrapper {
```
Comment (agadman) [inline style.css:53]: I like the way you work with Flexbox in the header.

**Review by agadman — APPROVED** [review]
Your portfolio look so nice! Clean and simple in the best way. Nice colors!
The project fulfills all basic requirements. It is responsive and the code works in different browsers.

The code is easy to follow, good naming on classes. I like the way you separate your code sections with comments. 

Great job!!

File: index.html (hunk 1 above)
Comment (petrasson) [reply index.html:55]: Yes, it's on the todo list 😅

File: index.html (hunk 3 above)
Comment (petrasson) [reply index.html]: Good point! I will add that to the todo list.

File: style.css (hunk 4 above)
Comment (petrasson) [reply style.css:53]: :-)

**Pull request closed without merging**